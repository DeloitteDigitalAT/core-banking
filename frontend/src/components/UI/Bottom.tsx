import React from "react";

const Bottom: React.FC = () => {
  return (
    <div className="px-4 text-gray-600 bg-gradient-to-b from-green-100 via-green-200 to-green-200 rounded text-sm py-5 mt-6 w-full">
      &copy; {new Date().getFullYear()} Deloitte Digital{" "}
    </div>
  );
};

export default Bottom;
