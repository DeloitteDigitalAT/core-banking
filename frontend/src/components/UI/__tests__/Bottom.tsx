import React from "react";
import renderer from "react-test-renderer";
import Bottom from "../Bottom";

describe("UI/Bottom component", () => {
  test("snapshot renders", () => {
    const tree = renderer.create(<Bottom />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
