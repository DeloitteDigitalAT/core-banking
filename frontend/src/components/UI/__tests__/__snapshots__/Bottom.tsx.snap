// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/Bottom component snapshot renders 1`] = `
<div
  className="px-4 text-gray-600 bg-gradient-to-b from-green-100 via-green-200 to-green-200 rounded text-sm py-5 mt-6 w-full"
>
  Â© 
  2021
   Deloitte Digital
   
</div>
`;
